### DISCUSSION TITLE
Exiftool 12.84+ now supports OpenTracks gpx speed, accuracy

### DISCUSSION THREAD

**Discussion by gitoss** [OpenTracksApp/OpenTracks#1901] [General]
Rejoice, the swiss army knife of metadata now reads OpenTracks' gpx export including speed and accuracy - and writes this information to the respective xmp metadata fields. https://exiftool.org/history.html

I had requested this feature, if there's still missing something feel free to join to request here: https://exiftool.org/forum/index.php?topic=15972.0

For a general q&a on how to geotag with exiftool (and geolocate) w/ exiftool: https://www.exiftool.org/geotag.html

**Comment by dennisguse** [comment]
Nice :)

Can you create a pull request for the README.md to add exiftool into the table?

**Comment by dennisguse** [comment]
Be aware that the accuracy_horizontal is the one provided by Androids LocationManager (aka radius in meter that has 65% likely to cover the real vs. measured location). 

For GPS there are other measures as well, but not exposed by Android.

**Comment by dennisguse** [comment]
Added to README: https://github.com/OpenTracksApp/OpenTracks/commit/a6f3c05e8262db6e93adcb6d1d2c635c25addc59
